packages/ui/src/components/Button/Button.stories.tsx

```diff
@@ -10,9 +10,9 @@ export default {
 } as Meta<typeof Button>
 
 export const ButtonVariants = () => {
-	const kind: any[] = ['primary', 'secondary']
-	const emphasis: any[] = ['high', 'medium', 'low']
-	const size: any[] = ['medium', 'small', 'xSmall']
+	const kind = ['primary', 'secondary'] as const
+	const emphasis = ['high', 'medium', 'low'] as const
+	const size = ['medium', 'small', 'xSmall'] as const
```

Comment:
These updates to the `*.stories.tsx` files are to fix some linting errors we are seeing Reflame comment on in our PRs. Not part of the original ticket.